angular.json

```diff
@@ -92,13 +92,13 @@
               "src/assets"
             ],
             "styles": [
-              "./node_modules/bootstrap/dist/css/bootstrap.min.css", 
+              "./node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
Do not see changes here

Reply:
IDK Why it went to pull. Not necessarily done by me. Probably GitHub issue, nothing to be rectified here.

Reply:
Yo that happened because you use a code formatter most likely. 
@KylixMedusa merge bhai 🤦‍♂